application/keycloak-third-party-authentication/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+name: keycloak-third-party-authentication
+services:
+  db-keycloak:
+    image: postgres:16
+    volumes:
+      - ./data/db-keycloak/postgresql-data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: postgres
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: keycloak
+  keycloak:
+    image: quay.io/keycloak/keycloak:23.0.7
+    volumes:
+      - ./data/keycloak/data:/opt/keycloak/data
+      - ./data/keycloak/themes:/opt/keycloak/themes
+      - ./data/keycloak/providers:/opt/keycloak/providers
```

Comment:
why do we not use volumes to mount these providers in our production keycloak also?